workspaces/adventure-pack/goodies/typescript/Iterator.prototype.take/index.test.ts

```diff
@@ -0,0 +1,89 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { iteratorPrototype } from "../Iterator.prototype";
+delete (iteratorPrototype as unknown as Record<string, unknown>).take;
+
+import "./index";
+
+function iteratorToArray<T>(iterator: IterableIterator<T>): T[] {
+  return [...iterator];
+}
```

Comment:
Does it work if we replace the callsites of this function with `Array.from`?

Reply:
it does